setup.py

```diff
@@ -1,25 +1,6 @@
 import glob
-import os
-
-from setuptools import setup, find_packages
-
-TRX_MINIMAL_INSTALL = os.environ.get('TRX_MINIMAL_INSTALL')
-TRX_MINIMAL_INSTALL = False if TRX_MINIMAL_INSTALL is None else \
-    int(TRX_MINIMAL_INSTALL)
-
-if TRX_MINIMAL_INSTALL:
-    SCRIPTS = None
-    SCRIPTS = glob.glob("scripts/tff_generate_trx_from_scratch.py")
-    REQUIRES = ['numpy>=1.20.*', 'nibabel>=3.*',
-                'pytest>=7.*', 'pytest-console-scripts>=0.*']
-else:
-    SCRIPTS = glob.glob("scripts/*.py")
-    REQUIRES = ['fury@git+https://github.com/frheault/fury.git@5059a529#egg=fury',
-                'dipy@git+https://github.com/frheault/dipy.git@4e192c5c6#egg=dipy',
-                'pytest>=7.*', 'pytest-console-scripts>=0.*']
+from setuptools import setup
 
 setup(name='trx',
-      packages=find_packages(),
-      setup_requires=['packaging>=19.0'],
-      install_requires=REQUIRES,
-      scripts=SCRIPTS)
+      use_scm_version=True,
```

Comment:
Curiosity:  This can not be added to `setup.cfg` or `pyproject.toml` ?

Reply:
Possibly. See [04ead56](https://github.com/tee-ar-ex/trx-python/pull/36/commits/04ead56c98dc7107aef6777b6ad3dd22c90fb434). 

However, I believe that we'd still need the setup.py file to flexibly install the scripts using the glob pattern.